doc/async_worker_variants.md

```diff
@@ -550,7 +550,7 @@ const onProgressCallback = (num) => {
     // ...
 };
 
-// Call our native addon with the paramters of a string and a function
+// Call our native addon with the paramters of a string three callback functions
```

Comment:
Maybe it shlould be:
`of a string and three callback functions`

Reply:
This is why I should not adjust my code late at night :( Fixing it